manage.py

```diff
@@ -12,6 +12,24 @@
 
 app = create_app()
 
+
+# gRPC server definitions, the dependencies required are not added to "requirements.txt" yet.
+# Added try-catch for this part to maintain usability of HTTP protocol without additional dependencies.
+# Dependencies: "grpcio-tools>=1.54.0", "grpcio-reflection>=1.54.0"
```

Comment:
Remember to add the dependencies to the txt when necessary or create an issue.

Reply:
#17